EmeraldIOS/Components/Selector/EmeraldSelectorField.swift

```diff
@@ -189,11 +188,18 @@ public class EmeraldSelectorField: EmeraldTextField, EmeraldSelectorFieldType, U
         inputView = pickerView
     }
 
+    private func shloulSetTextFirsTime() {
```

Comment:
Respecting this function name, it would must return a Bool, then, the name is incorrect, thus, rename it or change the logic. 
```suggestion
    private func shouldSetTextFirsTime() -> Bool {
```

Reply:
i rename the function by setupTextByFirsTime